### ISSUE TITLE
How to create .deb file of xdm with maven?

### ISSUE THREAD

**Issue by Andorlistick** [subhra74/xdm#183]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

Hello, having a 32 bit pc, I would like to know how to create a .deb file of xdm with maven. I tried all the methods but nothing worked. I would like to try the new improvements and not stay in version 2018.

**Comment by subhra74** [comment]
Were you able to create the jar file?
Maven will not create deb file. It will compile the project to executable jar.

**Comment by Andorlistick** [comment]
Yes but the java file doesn't work well. When I try to download a video it asks me to download the codecs and then I get an error.

**Comment by subhra74** [comment]
Please copy ffmpeg and youtube-dl binaries to that directory as well.
https://github.com/subhra74/xdm/wiki/Using-portable-XDM-jar

**Comment by Andorlistick** [comment]
It's working! Thank you very much for that.
